rbac/management/principal/proxy.py

```diff
@@ -282,7 +287,7 @@ def request_filtered_principals(self, principals, org_id=None, limit=None, offse
             return {"status_code": status.HTTP_200_OK, "data": []}
         filtered_principals_path = "/v1/users"
         params = self._create_params(limit, offset, options)
-        payload = {"users": principals, "include_permissions": False}
```

Comment:
this was not actually honoring the `include_permissions` field in the request to bop, so I updated it to work as intended